corehq/motech/repeaters/models.py

```diff
@@ -401,9 +401,8 @@ def _convert_to_serializable(self, repeater_dict):
     def get_url(self, record):
         return self.connection_settings.url
 
-    @classmethod
```

Comment:
I'm curious how you found this (and therefore how you know there aren't other instances of this pattern in HQ).

Reply:
I think I found this through a test failure, but I can't find that particular github action right now. It's possible it failed locally and I did not find it by running tests on github. I did some grepping to verify that there are no other instances with something like `ag @classmethod -C1 | grep @property | less`

Reply:
I forgot about `-C`, that's handy